pkg/chaining/pipeline.go

```diff
@@ -31,8 +33,38 @@ func (p *Pipeline) addStep(s service.Service, params *configuration.JiraAPIResou
 	p.steps = append(p.steps, newStep)
 }
 
-func (p *Pipeline) Execute() error {
+func (p *Pipeline) Execute(params *configuration.JiraAPIResourceParameters) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method Pipeline.Execute should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4778851)